pkg/operator/controller/ingress/metrics.go

```diff
@@ -26,9 +26,15 @@ var (
 		Help: "Report the conditions for ingress controllers. 0 is False and 1 is True.",
 	}, []string{"name", "condition"})
 
+	nlbUsage = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "ingress_controller_aws_nlb_usage",
```

Comment:
"usage" usually implies the amount of a limited resource (cpu usage, memory usage, etc.).  Is there a better name?  I would suggest *_count but you're not counting them.  It might be better to count the number in use over time.

Reply:
good point, I was just briefly looking through the others in cluster-monitoring-operator and came across "_info":
https://github.com/openshift/cluster-monitoring-operator/blob/master/jsonnet/rules.libsonnet#L343-L346

`ingress_controller_aws_nlb_info` maybe? wdyt? Or just dropping the suffix altogether.

Reply:
Maybe `ingress_controller_aws_nlb_active` if you don't count them.  But I still think it would be better to count them.

Reply:
> But I still think it would be better to count them.

gotta look up the Gauge implementation in Golang, but IIRC it will sum across all gauges along the "name" label. So for a cluster you should be able to get `sum(ingress_controller_aws_nlb_usage)` that would give you the number of active NLB ingress controllers, wouldn't it?

Or do you think we need to use a counter instead of a gauge?